Source/RxCocoa/UIScrollView+reachedBottom.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  UIScrollView+reachedBottom.swift
+//  RxSwiftExt
+//
+//  Created by Anton Nazarov on 09/05/2019.
+//  Copyright © 2019 RxSwift Community. All rights reserved.
+//
+
+import UIKit
+import RxSwift
+import RxCocoa
+
+public extension Reactive where Base: UIScrollView {
+    /**
+     Shows if the the bottom of the UIScrollView is reached.
+     - parameter offset: A threshhold indicating the bottom.
+     - returns: Sequence that emits when the bottom of the base UIScrollView is reached.
+     */
+    func reachedBottom(offset: CGFloat = 0.0) -> Signal<Void> {
+        return contentOffset.filterMap { contentOffset in
```

Comment:
don't forget `[weak base]`

Reply:
There is no need in it, no reference cycle here

Reply:
Not positive you're right, there might be a transitive cycle with `contentOffset`'s implementaion and `DelegateProxy`. Regardless, I'd never trust a "3rd party" implementation to provide retain-cycle safety for me. Unless I'm capturing the objective myself, the inner implementation could possibly "change under my feet". I'd add the capture group personally.